src/lib.rs

```diff
@@ -351,6 +354,13 @@ impl RegisterInfo {
     fn parse(tree: &Element) -> RegisterInfo {
         RegisterInfo {
             name: try!(tree.get_child_text("name")),
+            alternate_group: tree.get_child_text("alternateGroup"),
+            alternate_register: tree.get_child_text("alternateRegister"),
+            derived_from: tree.attributes.get("derivedFrom").map(
+                |s| {
+                    s.to_owned()
+                },
+            ),
```

Comment:
This indentation is uneeded

```rust
tree.attributes.get("derivedFrom").map(|s| s.to_owned()),
```

Reply:
I copied this from the other instance of `derivedFrom`.  Note that the `.rustfmt.toml` config wants this to be:

```
            derived_from: tree.attributes
                .get("derivedFrom")
                .map(|s| s.to_owned()),
```

There are other portions that are not rustfmt'd.   Would you like me to insert a commit that applies rustfmt underneath my changes and then make them match?